slack_sdk/rtm/v2/__init__.py

```diff
@@ -54,7 +56,7 @@ def __init__(
         timeout: int = 30,
         base_url: str = WebClient.BASE_URL,
         headers: Optional[dict] = None,
-        ping_interval: int = 10,
+        ping_interval: int = 5,
```

Comment:
adjusted the default value for more stable connection monitoring

Reply:
Looks like `node-slack-sdk` uses 6s as well:
https://github.com/slackapi/node-slack-sdk/blob/main/packages/rtm-api/src/KeepAlive.ts#L75-L76

Reply:
Thanks for sharing this! The existing RTMClient was using 30 seconds. I could make it a bit shorter 😄